src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tr');
+
+rows.forEach((row) => {
```

Comment:
Since `rows` is expected to be a NodeList (if `querySelectorAll` is used), you should use `Array.from(rows).forEach(...)` or spread it into an array `[...rows].forEach(...)` to iterate over it.